stdlib/3/asyncio/events.pyi

```diff
@@ -103,11 +103,18 @@ class AbstractEventLoop(metaclass=ABCMeta):
     @abstractmethod
     @coroutine
     def getnameinfo(self, sockaddr: tuple, flags: int = ...) -> Generator[Any, None, Tuple[str, int]]: ...
+    @overload
     @abstractmethod
     @coroutine
     def create_connection(self, protocol_factory: _ProtocolFactory, host: str = ..., port: int = ..., *,
-                          ssl: _SSLContext = ..., family: int = ..., proto: int = ..., flags: int = ..., sock: Optional[socket] = ...,
-                          local_addr: str = ..., server_hostname: str = ...) -> Generator[Any, None, _TransProtPair]: ...
+                          ssl: _SSLContext = ..., family: int = ..., proto: int = ..., flags: int = ..., sock: None = ...,
+                          local_addr: Optional[str] = ..., server_hostname: Optional[str] = ...) -> Generator[Any, None, _TransProtPair]: ...
+    @overload
+    @abstractmethod
+    @coroutine
+    def create_connection(self, protocol_factory: _ProtocolFactory, host: None = ..., port: None = ..., *,
+                          ssl: _SSLContext = ..., family: int = ..., proto: int = ..., flags: int = ..., sock: socket = ...,
```

Comment:
Shouldn't we remove the default value for `sock` in this case?

Reply:
I'm sorry — I don't understand what you mean by "remove the default value". In `sock: socket = ...` the default value is `Ellipsis` so I'd say it's already removed. I may be missing something obvious.

Reply:
I mean just writing `sock: socket`, without the `= ...`. That signifies that for this overload, the `sock` argument must be passed.

Reply:
OK. Thanks for the explanation, I didn't know that!

I made that change — and also rebased on top of master to fix conflicts.

Reply:
Also updated `create_server` for consistency. I had written it in the same style in #2763.